filebeat/_meta/config/filebeat.inputs.reference.yml.tmpl

```diff
@@ -439,7 +439,7 @@ filebeat.inputs:
 
   # Method to determine if two files are the same or not. By default
   # the Beat considers two files the same if their inode and device id are the same.
```

Comment:
Is this comment still true if the default method is now `fingerprint`?

Reply:
Thanks!